Runigram.java

```diff
@@ -60,48 +89,98 @@ private static void print(Color c) {
 	// For example, to check that some image processing function works correctly,
 	// we can apply the function and then use this function to print the resulting image.
 	private static void print(Color[][] image) {
-		//// Replace this comment with your code
+		for(int i=0; i<image.length; i++)
+		{
+			for(int j=0; j<image[0].length; j++)
+			{
+				System.out.print("(");
+				System.out.printf("%3s,",image[i][j].getRed());
+				System.out.printf("%3s,",image[i][j].getGreen());
+				System.out.printf("%3s",image[i][j].getBlue());
+				System.out.print(")  ");

```

Comment:
You should use the print method for Color type object.
```suggestion
				print(image[i][j]);

```